ruby/README.md

```diff
@@ -1,41 +1,25 @@
-# AWS SDK for Ruby - Code Examples
-[![Ruby Style Guide](https://img.shields.io/badge/code_style-rubocop-brightgreen.svg)](https://github.com/rubocop/rubocop)
-
+# AWS SDK for Ruby code examples
 ## Overview
+The code examples in this topic show you how to use the AWS SDK for Ruby with AWS. 
 
-The code examples in this topic show you how to use the AWS SDK for Ruby Version 3 
-with AWS. 
-
-The AWS SDK for Ruby Version 3 provides a Ruby API for AWS infrastructure services.
-Using the SDK, you can build applications on top of AWS services such as Amazon Simple 
-Storage Service (Amazon S3), Amazon Elastic Compute Cloud (Amazon EC2), and Amazon DynamoDB.
+The AWS SDK for Ruby provides a Ruby API for AWS infrastructure services. Using the SDK, you can build applications on top of Amazon S3, Amazon EC2, Amazon DynamoDB, and more.
 
 ## Types of code examples
-
 * **Single-service actions** - Code examples that show you how to call individual service functions.
 
 * **Single-service scenarios** - Code examples that show you how to accomplish a specific task by calling multiple functions within the same service.
 
 * **Cross-service examples** - Sample applications that work across multiple AWS services.
 
-### Finding code examples
+Single-service actions and scenarios are organized by AWS service in the [*example_code folder*](*link to example folder*). A README in each folder lists and describes how to run the examples.
```

Comment:
example_code folder should be a link:

`[example_code folder](example_code)`

Reply:
Done.